### PR TITLE
Admin Functionality to Landing Page

### PR DESCRIPTION
**Pull request by maxsebso**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/866744728/pulses/1191441203)

## This PR

- Added different landing page variation for an admin user
- This involves the following:
    - Adding School Directory and User Directory Link
    - Change Form History -> Data Insights 

## Screenshots

Screen Shot 2021-05-05 at 12 42 54 AM

### CODE REVIEW

File: src/containers/home/index.tsx [hunk 1]
```diff
@@ -122,40 +128,148 @@ const Home: React.FC = () => {
...
+              <Col span={12}>
+                <InContain
+                  onClick={() => {
+                    history.push(Routes.SETTINGS);
+                  }}
+                >
+                  <Row>
+                    <Col span={8}>
+                      <FolderOpenOutlined
+                        style={{
```
Comment (rymaju) [inline src/containers/home/index.tsx:142]: ⛏️ Might be worthwhile to put this in a styled component

File: src/containers/home/index.tsx [hunk 2]
```diff
@@ -122,40 +128,148 @@ const Home: React.FC = () => {
...
+                          <Paragraph>View and registered users</Paragraph>
+                        </Col>
+                      </Row>
+                    </Col>
+                  </Row>
+                </InContain>
+              </Col>
+            </Row>
+          ) : (
+            <></>
```
Comment (rymaju) [inline src/containers/home/index.tsx:201]: If the else of the ternary is empty can can we use an && instead?

File: src/containers/home/index.tsx [hunk 3]
```diff
@@ -122,40 +128,148 @@ const Home: React.FC = () => {
               </InContain>
             </Col>
           </Row>
+          {privilegeLevel === PrivilegeLevel.ADMIN ? (
```
Comment (rymaju) [inline src/containers/home/index.tsx:131]: 🔧 This doesnt need to be a ternary, we can make it an && instead
```suggestion
          {privilegeLevel === PrivilegeLevel.ADMIN && (
```
And then remove the empty react fragment we return if not an admin

**Review by rymaju — APPROVED** [review]
It works, although we can change the ternary to be a `&&` instead, also I would personally prefer using styled components instead of inline styles here.

File: src/containers/home/index.tsx (hunk 2 above)
Comment (maxsebso) [reply src/containers/home/index.tsx:204]: Yeeeee

File: src/containers/home/index.tsx (hunk 3 above)
Comment (maxsebso) [reply src/containers/home/index.tsx:134]: Bet

File: src/containers/home/index.tsx (hunk 1 above)
Comment (maxsebso) [reply src/containers/home/index.tsx:145]: Ohhh just got what you mean

Comment (maxsebso) [reply src/containers/home/index.tsx:145]: So I tried doing it with styled components and they turned out miniature

**Review by jackblanc — APPROVED** [review]

**Pull request merged**